source/clear-linux/get-started/bare-metal-install-desktop/bare-metal-install-desktop.rst

```diff
@@ -0,0 +1,447 @@
+feature that reports failures and crashes to the |CL| development
+team for improvements. For more information, see :ref:`telemetry-about`.
+
+#. From :guilabel:`Required Options`, select :guilabel:`Telemetry`.
+
+#. Select :kbd:`Confirm`.
+
+#. If you don't wish to participate, deselect :kbd:`Enable Telemetry`.
+
+#. Select :kbd:`Confirm`.
+
+   .. figure:: figures/bare-metal-install-desktop-11.png
+      :scale: 100%
+      :alt: Enable Telemetry
+
+      Figure 11: Enable Telemetry
+
+Advanced options
+****************
+
+After you complete the `Required options`_, we recommend completing
+:guilabel:`Advanced options`--though they're not required. Doing so
+customizes your development environment, so you're ready to go immediately
+after reboot.
+
+* `Bundle Selection`_
+
+* `Assign Hostname`_
+
+As for bundles, you can always add more later with :ref:`swupd-guide`.
+
+Bundle Selection
+================
+
+#. On the Advanced menu, select :guilabel:`Bundle Selection`
+
+#. Select your desired bundles.
+
+   .. figure:: figures/bare-metal-install-desktop-12.png
+      :scale: 100%
+      :alt: Bundle Selection
+
+      Figure 12: Bundle Selection
+
+#. Select :kbd:`Confirm`.
+
+#. View the bundles that you selected.
+
+   .. figure:: figures/bare-metal-install-desktop-13.png
+      :scale: 100%
+      :alt: Bundle Selections - Advanced Options
+
+      Figure 13: Bundle Selections - Advanced Options
+
+Optional: Skip to `Finish installation`_.
+
+Assign Hostname
+===============
+
+#. In Advanced Options, select :guilabel:`Assign Hostname`.
+
+#. In :guilabel:`Hostname`, enter the hostname only (excluding the domain).
+
+   .. figure:: figures/bare-metal-install-desktop-14.png
+      :scale: 100%
+      :alt: Assign Hostname
+
+      Figure 14: Assign Hostname
+
+   .. note::
+
+      Hostname does not allow empty spaces. Hostname must start with an
+      alphanumeric character but may also contain hyphens. Maximum length of
+      63 characters.
+
+#. Navigate to :kbd:`Confirm` until highlighted.
+
+#. Select :kbd:`Confirm`.
+
+Optional: Skip to `Finish installation`_.
+
+Finish installation
+*******************
+
+#. When you are satisfied with your installation configuration, select
+   :guilabel:`Install`.
+
+   .. figure:: figures/bare-metal-install-desktop-15.png
+      :scale: 100%
+      :alt: Assign Hostname
+
+      Figure 15: Finish installation
+
+   .. note:
+
+      All checkmarks must be visible in :guilabel:`Required Options` for you to select :guilabel:`Install`.
+
+#. If you do not enter a selection for all :guilabel:`Required Options`,
+   the :guilabel:`Install` button remains greyed out, as shown
+   in Figure 16. Return to `Required Options`_ and make selections.
+
+
+   .. figure:: figures/bare-metal-install-desktop-16.png
+      :scale: 100%
+      :alt: Required Options - Incomplete
+
+      Figure 16: Required Options - Incomplete
+
+#. After installation is complete, select :guilabel:`Exit`.
```

Comment:
@mdhorn Please see my latest commit. I simplified the wording here. While I'm aware the most people, like myself, will just reboot and remove the USB-- it may be better to be more explicit in the final steps. Thoughts?

Please let me know if you have any other comments; hoping to release this early today.